src/coreComponents/mesh/CMakeLists.txt

```
@@ -121,6 +121,16 @@ if( ENABLE_PAMELA )
     list( APPEND dependencyList PAMELA )
 endif()
 
+if( ENABLE_VTK )
+    message(STATUS "Adding VTKMeshGenerator sources and headers")
+    set( mesh_headers ${meshUtilities_headers} generators/VTKMeshGenerator.hpp )
+    set( mesh_headers ${meshUtilities_sources} generators/VTKMeshGenerator.cpp)
```

```suggestion
    set( mesh_headers ${mesh_headers} generators/VTKMeshGenerator.hpp )                                                                                                                                         
    set( mesh_sources ${mesh_sources} generators/VTKMeshGenerator.cpp)
```
@AntoineMazuyer This is the change I made to fix the undefined reference. The problem was coming from the fact that `meshUtilities_headers/sources` does not exist anymore. With this change I was able to compile on Lassen, and I assume this will fix the CI failures.